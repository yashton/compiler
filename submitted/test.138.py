"""  this is a 
triple quoted string
"""