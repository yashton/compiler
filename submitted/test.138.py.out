(LIT "")
(ERROR "Illegal character '"  this is a 
triple quoted string
"""' on line 1)
