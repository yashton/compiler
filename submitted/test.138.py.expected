(LIT "  this is a \r\ntriple quoted string\r\n")
(ENDMARKER)
